hack/commit-release.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+#
+# Copyright (c) 2021 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This generates the commit necessary for release of the next z-stream tag.
+
+current=$(rosa version)
```

Comment:
```suggestion
current=$(git describe --tags `git rev-list --tags --max-count=1`)
```

Should we instead base this off the repo in case someone doesn't have an updated binary?

Reply:
We might want a `git fetch --tags` before that.

Reply:
Good idea. Fixed now